### PR TITLE
Feat: Metis contract

### PR DESCRIPTION
**Pull request by yummyweb**
This should probably work. Will create MockDIAOracle.

### CODE REVIEW

**Review by githubdoramon — APPROVED** [review]

File: contracts/MWWRegistarMetis.sol [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.4 <0.9.0;
+
+import "./DIAOracleV2.sol";
+import "./MWWRegistarBase.sol";
+import "hardhat/console.sol";
```
Comment (githubdoramon) [inline contracts/MWWRegistarMetis.sol:6]: Remove if not being used

File: contracts/MWWRegistarMetis.sol [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+contract MWWRegistarMetis is MWWRegistarBase {
+	DIAOracleV2 public oracle;	
+
+    constructor(address[] memory acceptableTokenAddresses) MWWRegistarBase(acceptableTokenAddresses) {}
+
+	function setPriceFeed(address priceFeed) public onlyOwner {
```
Comment (githubdoramon) [inline contracts/MWWRegistarMetis.sol:13]: Worth to use some lint to format the code

**Review by githubdoramon — CHANGES_REQUESTED** [review]

File: contracts/MWWRegistarMetis.sol (hunk 1 above)
Comment (yummyweb) [reply contracts/MWWRegistarMetis.sol:6]: Hmm, yes I did not know consol.sol was being used or not.

Comment (yummyweb) [reply contracts/MWWRegistarMetis.sol:6]: And DIAOracleV2 is being used since it imports the contract.

**Comment by yummyweb** [comment]
Added solhint as a new linter, just run `yarn lint` worth updating the README as well to inform other developers.

**Review by githubdoramon — APPROVED** [review]

**Pull request merged**